File: Codeforces/Codeforces Round #787/C_Detective_Task.py
import sys 
input = sys.stdin.readline

t = int(input())
for _ in range(t):
    s = input().rstrip()
    suspects = 0

    first_zero = s.find('0')
    if first_zero != -1:
        suspects += 1
    else:
        first_zero = len(s)

    last_one = -1
    for i in range(len(s)-1, -1, -1):
        if s[i] == '1':
            last_one = i
            suspects += 1
            break

    suspects += first_zero - last_one - 1

    print(suspects)

File: Codeforces/Codeforces Round #787/C_Detective_Task.cpp
#include "bits/stdc++.h"
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int t;
    cin >> t;
    while(t--){
        string s;
        cin >> s;
        int answer = 0;
        int first_zero = s.find('0');

        if(first_zero != -1)
            answer += 1;
        else
            first_zero = s.length();
        
        int last_one = s.rfind('1');
        if(last_one != -1)
            answer += 1;
        
        answer += first_zero - last_one - 1;
        cout << answer << '\n';
    }

}
